### PR TITLE
feat(service-namespacing): Add better service namespacing

### PR DESCRIPTION
**Pull request by benjaminpaige**
## Purpose

Add appropriate name-spacing to any services that need more specificity. (compare/status) as well as any log groups that need more definition. 

#### Linked Issues to Close

https://qmacbis.atlassian.net/browse/OY2-23040

## Assorted Notes/Considerations/Learning

This looks like it deployed and updated the stack names but we should be mindful of merging these changes into master to ensure no data loss or problems.

### CODE REVIEW

**Review by jdinh8124 — APPROVED** [review]
This is awesome thank you!

File: serverless-compose.yml [hunk 1]
```diff
@@ -7,21 +7,21 @@ services:
...
   seatool:
     path: src/services/seatool
-  status:
-    path: src/services/status
+  mmdl-status:
+    path: src/services/mmdl-status
   appian-status:
     path: src/services/appian-status
-  compare:
+  compare-mmdl:
```
Comment (JonHolman) [inline serverless-compose.yml:14]: before this the changes I saw we're to `mmdl-compare`.  We should be consistent if the mmdl should go on the beginning or the end.

Comment (jdinh8124) [reply serverless-compose.yml:14]: This is a good point, I can change the appian namespaces over to whatever you land on @benjaminpaige

Comment (benjaminpaige) [reply serverless-compose.yml:14]: yeah thats a good point jon. id say lets do [the thing is does] +  [the thing it does it for] so we lets do `status-mmdl` & `status-appian` cool? i can make that change here

Comment (jdinh8124) [reply serverless-compose.yml:14]: I'm good with that

**Review by jdinh8124 — APPROVED** [review]

**Pull request merged**